docs/Makefile

```diff
@@ -112,5 +112,6 @@ validate-links:
 		-t ${antora_docker_image}:${antora_docker_image_tag} \
 		-c "cd /antora/${base_path} && find src -name '*.adoc' -print0 | xargs -0 -n1 asciidoc-link-check --progress --config config/validate-links.json"
 
+# TODO: we will need to include publication for spring module once we want it out
 deploy: clean managed
-	bin/deploy.sh --module ${module} --upstream ${upstream} --branch ${branch} ${sources}
+	bin/deploy.sh --module java --upstream ${upstream} --branch ${branch} ${sources}
```

Comment:
Declaration of `$module` was removed in https://github.com/lightbend/kalix-jvm-sdk/pull/1199

We'll need to publish once for each module once we add spring officially or do a larger refactor of the deploy script.